app/controllers/contacts_controller.rb

```diff
@@ -0,0 +1,16 @@
+class ContactsController < ApplicationController
+  def create
+    @contact = Contact.new(contact_params)
+    @contact.company_id = params[:company_id]
```

Comment:
same as above - how can you leverage your associations to create the relationship between a contact and a company?